test/Microsoft.AspNetCore.Server.KestrelTests/ChunkedRequestTests.cs

```diff
@@ -78,6 +78,8 @@ public static TheoryData<ServiceContext> ConnectionFilterData
                          "");
                     await connection.ReceiveEnd(
                         "HTTP/1.1 200 OK",
+                        "Date: " + testContext.DateHeaderValue,
+                        "Server: Kestrel",
```

Comment:
Also, should I go through and remove redundant calls to `response.Headers.Clear()` (used to clear the `Date` and `Server` headers)?